src/SprykerEco/Yves/Computop/Converter/InitEasyCreditConverter.php

```diff
@@ -22,8 +23,21 @@ protected function createResponseTransfer(array $decryptedArray, ComputopRespons
     {
         $responseTransfer = new ComputopEasyCreditInitResponseTransfer();
         $responseTransfer->fromArray($decryptedArray, true);
-        $responseTransfer->setHeader($header);
+        $responseTransfer->setHeader($this->updateResponseHeader($header));
 
         return $responseTransfer;
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\ComputopResponseHeaderTransfer $header
+     *
+     * @return \Generated\Shared\Transfer\ComputopResponseHeaderTransfer
+     */
+    protected function updateResponseHeader(ComputopResponseHeaderTransfer $header)
+    {
+        if ($header->getStatus() === ComputopConfig::AUTHORIZE_REQUEST_STATUS) {
```

Comment:
what about `$header->setIsSuccess($header->getStatus() === ComputopConfig::AUTHORIZE_REQUEST_STATUS);` ?

Reply:
EasyCredit has two success statuses instead of other payment methods that have only one success status "OK". This code has been added for update header if it's Init Request for EasyCredit.